baseplate/diagnostics/metrics.py

```diff
@@ -39,13 +39,16 @@ def on_server_span_created(self, context, server_span):
 class MetricsServerSpanObserver(SpanObserver):
     def __init__(self, batch, server_span):
         self.batch = batch
+        self.base_name = "server." + server_span.name
         self.timer = batch.timer("server." + server_span.name)
```

Comment:
now that `self.base_name` exists, it'd be good to use it here in the timer creation